packages/kolibri/components/pages/AppBarPage/internal/AppBar.vue

```diff
@@ -394,4 +421,15 @@
     font-size: 14px;
   }
 
+  // Hide top navbar, but keep it in the DOM for overflow calulations
```

Comment:
This is just a "i forgot" not an "it shouldn't be so" but what is the scenario where we have a hidden nav bar but still want it to be available for calculations? 

I'm also wondering if there are accessibility considerations, which is why I'm trying to remember the scenario